code/src/components/TaskList.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import tasks from 'reducers/tasks'
+
+const TaskItem = styled.article`
+display: flex;
+align-items: center;
+border-radius: 10px;
+padding: 3px;
+margin: 1rem 1.5rem;
+min-height: 4rem;
+position: relative;
+word-break: break-word;
+overflow-wrap: wrap;
```

Comment:
Great job here! I missed the word-break property on my own project, and only had overflows. Definitely need to be more mindful of this myself in the future and do some more cross-browser testing.